content-page.php

```diff
@@ -15,7 +15,13 @@
 		<?php do_action('vantage_entry_main_top') ?>
 
 		<header class="entry-header">
-			<h1 class="entry-title"><?php the_title(); ?></h1>
+			<?php if ( !is_page_template() ) { ?>
```

Comment:
The same goes in this section. We should be using the alternative control syntax.

http://php.net/manual/en/control-structures.alternative-syntax.php